projects/rero/ng-core/src/lib/record/search/record-search.component.ts

```diff
@@ -341,7 +342,8 @@ export class RecordSearchComponent implements OnInit, OnChanges, OnDestroy {
     if (this._config.resultsText) {
       return this._config.resultsText(this.hits);
     }
-    return this._translateService.stream('{{ total }} results', { total: this.total });
+    const total = this._recordService.totalHits(this.total);
+    return this._translateService.stream('{{ total }} results', { total });
```

Comment:
I think you can use `this.total` directly, as it contains already the final value.

Reply:
done